rest-tests/src/test/java/io/syndesis/qe/RestTestHooks.java

```diff
@@ -35,7 +35,7 @@ public void afterTest() {
     public void getLogs(Scenario scenario){
         if (scenario.isFailed()) {
             log.warn("Scenario {} failed, saving server logs and integration logs to scenario", scenario.getName());
-            scenario.embed(OpenShiftUtils.getInstance().getPodLog(OpenShiftUtils.getPodByPartialName("syndesis-server")).getBytes(), "text/plain");
+            scenario.embed(OpenShiftUtils.getInstance().getPodLog(OpenShiftUtils.getPodByPartialName("syndesis-server").get()).getBytes(), "text/plain");
```

Comment:
I wonder if there is a universe where this fails with NPE ^^

Reply:
probably there is, but the test failed anyway, so it will just fail with an error about getting logs from non-existing pod :dancer: